### PR TITLE
Add a style guide

### LINKED ISSUE #560: Add note about not translating the course title
The course title, "Comprehensive Rust 🦀", should be seen as a brand and not translated.

Add a note about this to TRANSLATIONS.md.

### PR DESCRIPTION
**Pull request by mgeisler**
This is an attempt at documenting some things to do and don’t do when updating the course text or a translation.

Fixes #560.

### CODE REVIEW

**Comment by mgeisler** [comment]
Hey Ivan, we talked about adding a style guide a while ago... I finally sat down and put something up for this :smile:

File: STYLE.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  (which means `[foo][foo]`), and `[foo]` (which also means `[foo][foo]`).
+
+  As an example, if you translate `[foo]`, to `[FOO]`, you must also update the
+  corresponding link definition from `[foo]: https://example.net` to `[FOO]:
+  https://example.net`. If you forget to do this, you end up with a broken link
+  in the translation.
+
+- If you find mistakes or things that sound awkward in the original English
+  text, please submit PRs to fix them! Fixing typos in the translation is great,
+  but we want evreybody to benefit from the fixes and that is why we need the
```
Comment (qwandor) [inline STYLE.md:89]: everybody

Comment (mgeisler) [reply STYLE.md]: Thanks!

**Review by qwandor — APPROVED** [review]

**Comment by mgeisler** [comment]
Thanks @qwandor!

**Pull request merged**